pkg/cmd/adm/register_member_test.go

```diff
@@ -459,47 +449,54 @@ func TestRegisterMember(t *testing.T) {
 		require.Error(t, err)
 		assert.Contains(t, term.Output(), "The toolchain-host-operator/toolchaincluster-host ServiceAccount is not present in the host cluster.")
 		tcs := &toolchainv1alpha1.ToolchainClusterList{}
-		require.NoError(t, fakeClient.List(context.TODO(), tcs, runtimeclient.InNamespace("toolchain-host-operator")))
+		require.NoError(t, fakeClient.List(context.TODO(), tcs, runtimeclient.InNamespace(hostNs)))
 		assert.Empty(t, tcs.Items)
-		require.NoError(t, fakeClient.List(context.TODO(), tcs, runtimeclient.InNamespace("toolchain-member-operator")))
+		require.NoError(t, fakeClient.List(context.TODO(), tcs, runtimeclient.InNamespace(memberNs)))
 		assert.Empty(t, tcs.Items)
 	})
 }
 
-func mockCreateToolchainClusterInNamespaceWithReadyCondition(fakeClient *test.FakeClient, namespace string) {
+func mockCreateToolchainClusterInNamespaceWithReadyCondition(t *testing.T, fakeClient *test.FakeClient, namespace string, hostOperatorNamespace, memberOperatorNamespace string) {
```

Comment:
are there any cases when
* the `hostOperatorNamespace` would be different than `test.HostOperatorNs`
* the `memberOperatorNamespace` would be different than `test.MemberOperatorNs`

? 
if not, then you could use the constants instead. 
The same applies for the other functions